data-fusion/s1_flooding_visualisation/copernicus_browser.js

```diff
@@ -0,0 +1,36 @@
+//VERSION=3
+function setup() {
+    return {
+      input: [{
+          datasource: "S2L2A",
+          bands: ["B08", "B03", "B04"]
```

Comment:
The B03 and B04 band are not accessed, which can be removed.

Reply:
done